lib/CMakeLists.txt

```
@@ -188,11 +188,14 @@ if(GRN_WITH_MESSAGE_PACK)
   endif()
 endif()
 if(GRN_WITH_LLAMA_CPP)
```

Does this work? Do we need `if(GRN_WITH_LLAMA_CPP) if(GRN_WITH_LLAMA_CPP_BUNDLED)`?

```suggestion
if(GRN_WITH_LLAMA_CPP_BUNDLED)
```

---

fix: [1261951](https://github.com/groonga/groonga/pull/2289/commits/12619512f682290fa095d5a8b84854082cf341f3) We don't need the checking for `GRN_WITH_LLAMA_CPP` here.

> Does this work?

This change works as follows although expected value aren't matched and there are some warnings.
I confirmed it on macOS Sonoma 14.5 with M3 chip.
```console
$ cmake -S . -B ../groonga.build --preset=debug-default -DCMAKE_INSTALL_PREFIX=/tmp/local -DGRN_WITH_LLAMA_CPP=system
$ cmake --build ../groonga.build
$ cmake --install ../groonga.build
/tmp/local/bin/groonga --versionGroonga 15.0.4-26-g0f8971c [Darwin,arm64,utf8,match-escalation-threshold=0,nfkc,mecab,message-pack,onigmo,zlib,lz4,zstandard,kqueue,xxhash,h3,simdjson,llama.cpp]
$ grntest --groonga=/tmp/local/bin/groonga test/command/suite/select/function/language_model_vectorize/function.test
  tests/sec |    tests |   passes | failures |   leaked |  omitted | !checked |
|>                                                                                                                                                                           | [  0%]
=====================================================================================================================================================================================
[0] .
  function                                                                                                                                                          23.0860s [failed]
=====================================================================================================================================================================================
--- (expected)
+++ (actual)
@@ -37,19 +37,71 @@
       [
         "I am a king.",
         [
-          -0.009299132,
-          -0.007786971,
-          0.000646766
+          -0.008953058,
+          -0.007886766,
+          5.646741e-4
         ]
       ],
       [
         "I am a queen.",
         [
-          -0.01004994,
-          -0.003906215,
-          -0.001882675
+          -0.009819536,
+          -0.003987025,
+          -0.002043356
         ]
       ]
     ]
   ]
 ]
+#|w| ggml_metal_init: skipping kernel_get_rows_bf16                     (not supported)
+#|w| ggml_metal_init: skipping kernel_mul_mv_bf16_f32                   (not supported)
+#|w| ggml_metal_init: skipping kernel_mul_mv_bf16_f32_1row              (not supported)
+#|w| ggml_metal_init: skipping kernel_mul_mv_bf16_f32_l4                (not supported)
+#|w| ggml_metal_init: skipping kernel_mul_mv_bf16_bf16                  (not supported)
+#|w| ggml_metal_init: skipping kernel_mul_mv_id_bf16_f32                (not supported)
+#|w| ggml_metal_init: skipping kernel_mul_mm_bf16_f32                   (not supported)
+#|w| ggml_metal_init: skipping kernel_mul_mm_id_bf16_f32                (not supported)
+#|w| ggml_metal_init: skipping kernel_flash_attn_ext_bf16_h64           (not supported)
+#|w| ggml_metal_init: skipping kernel_flash_attn_ext_bf16_h80           (not supported)
+#|w| ggml_metal_init: skipping kernel_flash_attn_ext_bf16_h96           (not supported)
+#|w| ggml_metal_init: skipping kernel_flash_attn_ext_bf16_h112          (not supported)
+#|w| ggml_metal_init: skipping kernel_flash_attn_ext_bf16_h128          (not supported)
+#|w| ggml_metal_init: skipping kernel_flash_attn_ext_bf16_h192          (not supported)
+#|w| ggml_metal_init: skipping kernel_flash_attn_ext_bf16_hk192_hv128   (not supported)
+#|w| ggml_metal_init: skipping kernel_flash_attn_ext_bf16_h256          (not supported)
+#|w| ggml_metal_init: skipping kernel_flash_attn_ext_bf16_hk576_hv512   (not supported)
+#|w| ggml_metal_init: skipping kernel_flash_attn_ext_vec_bf16_h96       (not supported)
+#|w| ggml_metal_init: skipping kernel_flash_attn_ext_vec_bf16_h128      (not supported)
+#|w| ggml_metal_init: skipping kernel_flash_attn_ext_vec_bf16_h192      (not supported)
+#|w| ggml_metal_init: skipping kernel_flash_attn_ext_vec_bf16_hk192_hv128 (not supported)
+#|w| ggml_metal_init: skipping kernel_flash_attn_ext_vec_bf16_h256      (not supported)
+#|w| ggml_metal_init: skipping kernel_flash_attn_ext_vec_bf16_hk576_hv512 (not supported)
+#|w| ggml_metal_init: skipping kernel_cpy_f32_bf16                      (not supported)
+#|w| ggml_metal_init: skipping kernel_cpy_bf16_f32                      (not supported)
+#|w| ggml_metal_init: skipping kernel_cpy_bf16_bf16                     (not supported)
+#|w| ggml_metal_init: skipping kernel_get_rows_bf16                     (not supported)
+#|w| ggml_metal_init: skipping kernel_mul_mv_bf16_f32                   (not supported)
+#|w| ggml_metal_init: skipping kernel_mul_mv_bf16_f32_1row              (not supported)
+#|w| ggml_metal_init: skipping kernel_mul_mv_bf16_f32_l4                (not supported)
+#|w| ggml_metal_init: skipping kernel_mul_mv_bf16_bf16                  (not supported)
+#|w| ggml_metal_init: skipping kernel_mul_mv_id_bf16_f32                (not supported)
+#|w| ggml_metal_init: skipping kernel_mul_mm_bf16_f32                   (not supported)
+#|w| ggml_metal_init: skipping kernel_mul_mm_id_bf16_f32                (not supported)
+#|w| ggml_metal_init: skipping kernel_flash_attn_ext_bf16_h64           (not supported)
+#|w| ggml_metal_init: skipping kernel_flash_attn_ext_bf16_h80           (not supported)
+#|w| ggml_metal_init: skipping kernel_flash_attn_ext_bf16_h96           (not supported)
+#|w| ggml_metal_init: skipping kernel_flash_attn_ext_bf16_h112          (not supported)
+#|w| ggml_metal_init: skipping kernel_flash_attn_ext_bf16_h128          (not supported)
+#|w| ggml_metal_init: skipping kernel_flash_attn_ext_bf16_h192          (not supported)
+#|w| ggml_metal_init: skipping kernel_flash_attn_ext_bf16_hk192_hv128   (not supported)
+#|w| ggml_metal_init: skipping kernel_flash_attn_ext_bf16_h256          (not supported)
+#|w| ggml_metal_init: skipping kernel_flash_attn_ext_bf16_hk576_hv512   (not supported)
+#|w| ggml_metal_init: skipping kernel_flash_attn_ext_vec_bf16_h96       (not supported)
+#|w| ggml_metal_init: skipping kernel_flash_attn_ext_vec_bf16_h128      (not supported)
+#|w| ggml_metal_init: skipping kernel_flash_attn_ext_vec_bf16_h192      (not supported)
+#|w| ggml_metal_init: skipping kernel_flash_attn_ext_vec_bf16_hk192_hv128 (not supported)
+#|w| ggml_metal_init: skipping kernel_flash_attn_ext_vec_bf16_h256      (not supported)
+#|w| ggml_metal_init: skipping kernel_flash_attn_ext_vec_bf16_hk576_hv512 (not supported)
+#|w| ggml_metal_init: skipping kernel_cpy_f32_bf16                      (not supported)
+#|w| ggml_metal_init: skipping kernel_cpy_bf16_f32                      (not supported)
+#|w| ggml_metal_init: skipping kernel_cpy_bf16_bf16                     (not supported)
=====================================================================================================================================================================================
  tests/sec |    tests |   passes | failures |   leaked |  omitted | !checked |
|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------| [100%]

  tests/sec |    tests |   passes | failures |   leaked |  omitted | !checked |
       0.04 |        1 |        0 |        1 |        0 |        0 |        0 |
0% passed in 23.0919s.

```